syschdemd.sh

```diff
@@ -38,6 +38,6 @@ if [[ $# -gt 0 ]]; then
     shift
     cmd="$@"
 else
-    cmd="$userShell"
+    cmd="$userShell --login"
```

Comment:
The short flag `-l` is more portable than `--login`, which doesn't work for dash or mksh, for example.